i18n/ja/docusaurus-plugin-content-docs/current/unraid-os/release-notes/6.12.8.md

```
@@ -1,7 +1,8 @@
 # バージョン 6.12.8 2024-02-15
 
-## アップグレードに関する注意事項 {#upgrade-notes}
+<span id="upgrade-notes"></span>
 
+## アップグレードに関する注意事項
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)